src/tests/system/tests/test_autofs.py

```diff
@@ -22,7 +22,7 @@
     lambda client, sssd_service_user: ((sssd_service_user == "root") or client.features["non-privileged"]),
     "SSSD was built without support for running under non-root",
 )
-def test_autofs__cache_first(
+def test_autofs__cache_first_set_to_true(
```

Comment:
I changed it to autofs, since most of our tests are named after the service/tool verbatim instead of using the full project name. test_system_security_services_daemon_automount_file_system_on_red_hat_enterprise_linux_for_ibm_s390x_virtual_machines... now there is a point that too long is just well too damn long.